### ISSUE TITLE
ERROR:tornado.general:Could not open static file '/base/images/favicon.ico'

### ISSUE THREAD

**Issue by wanted68** [QuantStack/voila#56]
I installed voila but when I run a notebook and want to access to it I get this error:

```
`ERROR:tornado.application:Uncaught exception GET / (127.0.0.1)
HTTPServerRequest(protocol='http', host='localhost:8300', method='GET', uri='/', version='HTTP/1.1', remote_ip='127.0.0.1')
Traceback (most recent call last):
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/nbconvert/preprocessors/execute.py", line 423, in _wait_for_reply
    msg = self.kc.shell_channel.get_msg(timeout=timeout)
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/jupyter_client/blocking/channels.py", line 57, in get_msg
    raise Empty
queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/tornado/web.py", line 1592, in _execute
    result = yield result
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/tornado/gen.py", line 1133, in run
    value = future.result()
  File "/usr/lib/python3.5/asyncio/futures.py", line 274, in result
    raise self._exception
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/tornado/gen.py", line 1147, in run
    yielded = self.gen.send(value)
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/voila/handler.py", line 53, in get
    result = executenb(notebook, km=km)
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/nbconvert/preprocessors/execute.py", line 541, in executenb
    return ep.preprocess(nb, resources, km=km)[0]
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/nbconvert/preprocessors/execute.py", line 361, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/nbconvert/preprocessors/base.py", line 69, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/nbconvert/preprocessors/execute.py", line 376, in preprocess_cell
    reply, outputs = self.run_cell(cell, cell_index)
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/nbconvert/preprocessors/execute.py", line 447, in run_cell
    exec_reply = self._wait_for_reply(msg_id, cell)
  File "/home/ubuntu/.newenv/lib/python3.5/site-packages/nbconvert/preprocessors/execute.py", line 436, in _wait_for_reply
    raise exception("Cell execution timed out")
TimeoutError: Cell execution timed out
ERROR:tornado.general:Could not open static file '/base/images/favicon.ico'
ERROR:tornado.general:Could not open static file '/style/style.min.css'
ERROR:tornado.access:500 GET / (127.0.0.1) 47190.06ms
`
```

**Comment by maartenbreddels** [comment]
It seems more like an nbconvert issue, which version are you using?

**Comment by wanted68** [comment]
`Name: nbconvert
Version: 5.4.0
Summary: Converting Jupyter Notebooks
Home-page: https://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: BSD
Location: /home/ubuntu/.newenv/lib/python3.5/site-packages
Requires: pygments, jinja2, entrypoints, traitlets, defusedxml, bleach, mistune, nbformat, testpath, jupyter-core, pandocfilters
Required-by: notebook, nbinteract, jupyter-server`

**Comment by HashRocketSyntax** [comment]
Same errors.

```
Name: nbconvert
Version: 5.5.0

Name: voila
Version: 0.1.3
```

With the addition of this:
`AssertionError: wrong color format 'var(--jp-mirror-editor-variable-color)'`

**Comment by HashRocketSyntax** [comment]
![image](https://user-images.githubusercontent.com/16389820/60525382-38120300-9cbc-11e9-9a7a-f4d0249e58c9.png)
> https://buildmedia.readthedocs.org/media/pdf/tornado/branch3.2/tornado.pdf

**Comment by HashRocketSyntax** [comment]
This seems like a duplicate of https://github.com/QuantStack/voila/issues/56

**Comment by maartenbreddels** [comment]
I'm assuming this was fixed, since #224 is closed, and the 404 was unrelated I think. If not, please reopen.

**Comment by farmani60** [comment]
I am using:

nbconvertor: 5.6.0
voila: 0.1.9

and also getting this error:


[Voila] ERROR | Timeout waiting for execute reply (30s).
ERROR:tornado.application:Uncaught exception GET / (::1)
HTTPServerRequest(protocol='http', host='localhost:8866', method='GET', uri='/', version='HTTP/1.1', remote_ip='::1')
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1592, in _execute
    result = yield result
  File "/usr/local/lib/python2.7/dist-packages/tornado/gen.py", line 1133, in run
    value = future.result()
  File "/usr/local/lib/python2.7/dist-packages/tornado/concurrent.py", line 261, in result
    raise_exc_info(self._exc_info)
  File "/usr/local/lib/python2.7/dist-packages/tornado/gen.py", line 1147, in run
    yielded = self.gen.send(value)
  File "/usr/local/lib/python2.7/dist-packages/voila/handler.py", line 55, in get
    result = executenb(notebook, km=km, cwd=cwd, config=self.traitlet_config)
  File "/usr/local/lib/python2.7/dist-packages/voila/execute.py", line 136, in executenb
    return ep.preprocess(nb, resources, km=km)[0]
  File "/usr/local/lib/python2.7/dist-packages/voila/execute.py", line 89, in preprocess
    result = super(VoilaExecutePreprocessor, self).preprocess(nb, resources=resources, km=km)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/execute.py", line 405, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/base.py", line 69, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/execute.py", line 438, in preprocess_cell
    reply, outputs = self.run_cell(cell, cell_index, store_history)
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/execute.py", line 571, in run_cell
    if self._passed_deadline(deadline):
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/execute.py", line 541, in _passed_deadline
    self._handle_timeout()
  File "/usr/local/lib/python2.7/dist-packages/nbconvert/preprocessors/execute.py", line 504, in _handle_timeout
    raise TimeoutError("Cell execution timed out")
RuntimeError: Cell execution timed out
ERROR:tornado.general:Could not open static file '/base/images/favicon.ico'
ERROR:tornado.general:Could not open static file '/style/style.min.css'
ERROR:tornado.access:500 GET / (::1) 38847.19ms

**Comment by maartenbreddels** [comment]
> RuntimeError: Cell execution timed out

This is your problem, it times out, it has nothing to do with the favicon. Does you cell take long to run? See https://github.com/QuantStack/voila/pull/240 for how to fix this.

I think we should document this.

**Comment by farmani60** [comment]
Can it be something wrong with my browser? It also gives this:

ERROR:browser_process_sub_thread.cc(203)] Waited 7 ms for network service

**Comment by maartenbreddels** [comment]
I don't think something is wrong with your browser, they always try to fetch the favicon. That error msg gives some google hits, but these are totally unrelated to the timeout.

**Comment by farmani60** [comment]
Thanks just ran it by specifying:

--ExecutePreprocessor.timeout=N

**Comment by jtpio** [comment]
Maybe we could also document how to "hide" long and expensive computation behind buttons or interactive components.
